src/main/java/ladder/model/Ladder.java

```diff
@@ -0,0 +1,32 @@
+package ladder.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import static ladder.model.Line.createOneLine;
+
+public class Ladder {
+    private List<Line> lines = new ArrayList<>();
+
+    public Ladder(int heightOfLadder, int NumberOfName) {
```

Comment:
변수명 [컨벤션](https://google.github.io/styleguide/javaguide.html#s5.2.6-parameter-names) 확인 부탁드려요
```suggestion
    public Ladder(int heightOfLadder, int numberOfName) {
```